inferentia/scripts/setup.sh

```diff
@@ -28,28 +28,31 @@
 USAGE="
 usage: test_script.sh [options]
 
-Sets up enviroment for neuron-pytorch
+Sets up enviroment for Inferentia-neuron
```

Comment:
Sets up python execution environment for AWS Neuron SDK for execution on Inferentia chips.